docs: clean up starter flows and screenshots (#7428)
langflow-ai/langflow: docs/docs/Starter-Projects/starter-projects-simple-agent.md

## Prerequisites

> - [Langflow installed and running](/get-started-installation)
> - [An OpenAI API key created](https://platform.openai.com/)

Reviewer: ```suggestion
- [A running Langflow instance](/get-started-installation)
- [An OpenAI API key](https://platform.openai.com/)
```
